scripts/run_performance_job.py

```diff
@@ -0,0 +1,517 @@
+                "call %HELIX_WORKITEM_PAYLOAD%\\.venv\\Scripts\\activate.bat",
+                "set PYTHONPATH=",
+                "py -3 -m pip install azure.storage.blob==12.0.0",
+                "py -3 -m pip install azure.storage.queue==12.0.0",
+                "py -3 -m pip install urllib3==1.26.15 --force-reinstall",
+                f"set \"PERFLAB_UPLOAD_TOKEN={args.perflab_upload_token}\"",
+                *additional_helix_pre_commands
+            ]
+        else:
+            helix_pre_commands = [
+                preserve_python_path,
+                "export CRYPTOGRAPHY_ALLOW_OPENSSL_102=true"
+                "sudo apt-get -y install python3-venv",
+                "python3 -m venv $HELIX_WORKITEM_PAYLOAD/.venv",
+                ". $HELIX_WORKITEM_PAYLOAD/.venv/bin/activate",
+                "export PYTHONPATH=",
+                "python3 -m pip install -U pip",
+                "pip3 install azure.storage.blob==12.0.0",
+                "pip3 install azure.storage.queue==12.0.0",
+                "pip3 install urllib3==1.26.15 --force-reinstall",
+                f"export PERFLAB_UPLOAD_TOKEN=\"{args.perflab_upload_token}\"",
+                *additional_helix_pre_commands
+            ]
+    else:
+        if args.os_group == "windows":
+            helix_pre_commands = [
+                "set ORIGPYPATH=%PYTHONPATH%",
+                "py -m pip install -U pip",
+                "py -3 -m venv %HELIX_WORKITEM_PAYLOAD%\\.venv",
+                "call %HELIX_WORKITEM_PAYLOAD%\\.venv\\Scripts\\activate.bat",
+                "set PYTHONPATH=",
+                "py -3 -m pip install -U pip",
+                "py -3 -m pip install --user urllib3==1.26.15 --force-reinstall",
+                "py -3 -m pip install --user azure.storage.blob==12.0.0 --force-reinstall",
+                "py -3 -m pip install --user azure.storage.queue==12.0.0 --force-reinstall",
+                f'set "PERFLAB_UPLOAD_TOKEN={args.perflab_upload_token}"'
+            ]
+        elif args.os_sub_group == "_musl":
+            helix_pre_commands = [
+                "export ORIGPYPATH=$PYTHONPATH",
+                "sudo apk add icu-libs krb5-libs libgcc libintl libssl1.1 libstdc++ zlib cargo",
+                "sudo apk add libgdiplus --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing",
+                "python3 -m venv $HELIX_WORKITEM_PAYLOAD/.venv",
+                "source $HELIX_WORKITEM_PAYLOAD/.venv/bin/activate",
+                "export PYTHONPATH=",
+                "python3 -m pip install -U pip",
+                "pip3 install --user urllib3==1.26.15 --force-reinstall",
+                "pip3 install --user azure.storage.blob==12.7.1 --force-reinstall",
+                "pip3 install --user azure.storage.queue==12.1.5 --force-reinstall",
+                f'export PERFLAB_UPLOAD_TOKEN="{args.perflab_upload_token}"'
+            ]
+        else:
+            if args.runtime_type == "wasm":
+                wasm_precommand = (
```

Comment:
This needs to be synced with https://github.com/dotnet/runtime/pull/90093 and https://github.com/dotnet/runtime/pull/90113.

Reply:
Done